io.edgehog.devicemanager.MobileConnectionInfo.json

```diff
@@ -0,0 +1,25 @@
+{
+  "interface_name": "io.edgehog.devicemanager.MobileConnectionInfo",
+  "version_major": 1,
+  "version_minor": 0,
+  "type": "properties",
+  "ownership": "device",
+  "description": "General hardware capabilities",
+  "mappings": [
+    {
+      "endpoint": "/apn",
```

Comment:
maybe we should make it parametric, when dealing with more than one modem. Anyway I'm not sure if we are already ready to add modem info now.
@rbino: any thought?